backend/shigai-serve/src/main.rs

```diff
@@ -0,0 +1,127 @@
+use std::io::prelude::{Read, Write};
+use std::net::TcpListener;
+use std::net::TcpStream;
+
+const SRC_ROOT: &str = "../../frontend/src";
+
+fn main() {
+    let listener = TcpListener::bind("127.0.0.1:7878").unwrap();
```

Comment:
Obviously this will need to be parameterized. I imagine we'll probably want to read in configuration details from an external file, and we can have testing and production profiles.

Reply:
leave for future work